packages/teleport/src/services/audit/makeEvent.ts

```diff
@@ -41,6 +41,12 @@ export const formatters: Formatters = {
     desc: 'Access Request Deleted',
     format: ({ id }) => `Access request [${id}] has been deleted`,
   },
+  [eventCodes.ACCESS_REQUEST_RESOURCE_SEARCH]: {
+    type: 'access_request.search',
+    desc: 'Access Request Resource Searched',
```

Comment:
```suggestion
    desc: 'Resource Access Request Search',
```
Hard to come up with a good wording for this but here's my suggestion. I think we are going to be calling these "Resource Access Requests" so this might be a bit more consistent